themes/default/views/product.blade.php

```diff
@@ -37,28 +37,33 @@ class="@if ($category->name == $categoryItem->name) text-secondary-900 pl-3 !bor
                     <p>{{ $category->description }}</p>
                 </div>
                 <div class="grid grid-cols-3 gap-4 mt-4">
-                    @foreach ($category->products()->orderBy('order')->get() as $product)
```

Comment:
Please revert this change

Reply:
it wont work if i didn't do it like this
it well always show out of stock